src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.AuthenticationRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import mate.academy.spring.service.dto.mapping.DtoResponseMapper;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final DtoResponseMapper<UserResponseDto, User> dtoResponseMapper;
+
+    public AuthenticationController(AuthenticationService authenticationService,
+                                    DtoResponseMapper<UserResponseDto, User> dtoResponseMapper) {
+        this.authenticationService = authenticationService;
+        this.dtoResponseMapper = dtoResponseMapper;
+    }
+
+    @PostMapping("/register")
+    public UserResponseDto register(@RequestBody AuthenticationRequestDto requestDto) {
```

Comment:
i guess AuthenticationRequestDto is not a croorect name to class. Rename class to UserRequestDto, because we should accept user and register him through service.